lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -1,5 +1,12 @@
 require 'tmpdir'
-require 'RMagick'
+begin
+  skip_magick = false
+  require 'RMagick'
+rescue LoadError
+  puts "Please, install RMagick if you aim to generate the PromoScreenshots."
+  puts "\'bundle install --with screenshots\' should do it if your project is configured for PromoScreenshots."
```

Comment:
Is this error printed whenever any `fastlane` task is run?

Reply:
Yes, it is. It's printed every time the plugin is loaded, which means every time `fastlane` is run. 
I took a  _very quick_ look at fastlane code and it doesn't seem possible to understand what action has been called at this time. 

Though, now that I re-think at this, we could try to only set the flag here, with no error message, and show the error message  when the action is called.
Wdyt?

Reply:
Yeah, I think it would be nice if the error was only when the promo screenshots were run. It should be easy to move this to `initialize` using the `skip_magick` variable.

Reply:
Changes here: https://github.com/wordpress-mobile/release-toolkit/pull/63